### PR TITLE
CLOUDP-225156 Remove github.com/containers/podman dependency

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Remove github.com/containers/podman dependency

_Jira ticket:_ CLOUDP-225156

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by tibulca — DISMISSED** [review]

File: internal/podman/config.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 MongoDB Inc
```
Comment (gssbzn) [inline internal/podman/config.go:1]: I suppose this is Podman?

Comment (fmenezes) [reply internal/podman/config.go:1]: this license is ours, their files are not stamped with licenses but looking at https://github.com/containers/podman/blob/main/LICENSE it seems the same

File: internal/podman/network.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+// limitations under the License.
+
+package podman
+
+import (
+	"net"
+	"time"
+)
+
+// file partially copied from https://github.com/containers/common/blob/v0.56.0/libnetwork/types/network.go
```
Comment (gssbzn) [inline internal/podman/network.go:22]:
```suggestion
// file partially inspired by https://github.com/containers/common/blob/v0.56.0/libnetwork/types/network.go
```

File: internal/podman/docker_schema2.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podman
+
+import "time"
+
+// file partially copied from https://github.com/containers/image/blob/v5.28.0/manifest/docker_schema2.go#L44-L64
```
Comment (gssbzn) [inline internal/podman/docker_schema2.go:19]:
```suggestion
// file partially inspired by https://github.com/containers/image/blob/v5.28.0/manifest/docker_schema2.go#L44-L64
```

**Review by gssbzn — APPROVED** [review]
LGTM

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by gssbzn — APPROVED** [review]

**Review by tibulca — APPROVED** [review]

**Pull request merged**